diff --git a/.gitignore b/.gitignore
new file mode 100644
index 0000000..479e0b6
--- /dev/null
+++ b/.gitignore
@@ -0,0 +1,4 @@
+config.h
+st.o
+x.o
+st
diff --git a/Makefile b/Makefile
index 470ac86..c4207b2 100644
--- a/Makefile
+++ b/Makefile
@@ -31,6 +31,7 @@ st: $(OBJ)
 
 clean:
 	rm -f st $(OBJ) st-$(VERSION).tar.gz
+	rm config.h
 
 dist: clean
 	mkdir -p st-$(VERSION)
diff --git a/config.def.h b/config.def.h
index 91ab8ca..9a91b75 100644
--- a/config.def.h
+++ b/config.def.h
@@ -5,7 +5,7 @@
  *
  * font: see http://freedesktop.org/software/fontconfig/fontconfig-user.html
  */
-static char *font = "Liberation Mono:pixelsize=12:antialias=true:autohint=true";
+static char *font = "Liberation Mono:pixelsize=24:antialias=true:autohint=true";
 static int borderpx = 2;
 
 /*
@@ -95,25 +95,25 @@ unsigned int tabspaces = 8;
 
 /* Terminal colors (16 first used in escape sequence) */
 static const char *colorname[] = {
-	/* 8 normal colors */
-	"black",
-	"red3",
-	"green3",
-	"yellow3",
-	"blue2",
-	"magenta3",
-	"cyan3",
-	"gray90",
-
-	/* 8 bright colors */
-	"gray50",
-	"red",
-	"green",
-	"yellow",
-	"#5c5cff",
-	"magenta",
-	"cyan",
-	"white",
+	/* 8 normal colours */
+	"#282828", /* hard contrast: #1d2021 / soft contrast: #32302f */
+	"#ea6962", /* red     */
+	"#a9b665", /* green   */
+	"#d8a657", /* yellow  */
+	"#7daea3", /* blue    */
+	"#d3869b", /* magenta */
+	"#89b482", /* cyan    */
+	"#d4be98", /* white   */
+
+	/* 8 light colours */
+	"#928374", /* black   */
+	"#ef938e", /* red     */
+	"#bbc585", /* green   */
+	"#e1bb7e", /* yellow  */
+	"#9dc2ba", /* blue    */
+	"#e1acbb", /* magenta */
+	"#a7c7a2", /* cyan    */
+	"#e2d3ba", /* white   */
 
 	[255] = 0,
 
@@ -129,8 +129,8 @@ static const char *colorname[] = {
  * Default colors (colorname index)
  * foreground, background, cursor, reverse cursor
  */
-unsigned int defaultfg = 258;
-unsigned int defaultbg = 259;
+unsigned int defaultfg = 15;
+unsigned int defaultbg = 0;
 unsigned int defaultcs = 256;
 static unsigned int defaultrcs = 257;
 
